src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -224,17 +227,19 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak, åp
                             </Alertstripe>
                         ) : (
                             <VedtaksbegrunnelseTeksterProvider>
-                                <VedtakBegrunnelserProvider
-                                    fagsak={fagsak}
-                                    aktivVedtak={aktivVedtak}
-                                >
-                                    <VedtakBegrunnelser åpenBehandling={åpenBehandling} />
-                                    <AvslagBegrunnelser åpenBehandling={åpenBehandling} />
-                                </VedtakBegrunnelserProvider>
-
+                                {!brukNyeVedtaksperioder && (
+                                    <VedtakBegrunnelserProvider
+                                        fagsak={fagsak}
+                                        aktivVedtak={aktivVedtak}
+                                    >
+                                        <VedtakBegrunnelser åpenBehandling={åpenBehandling} />
+                                        <AvslagBegrunnelser åpenBehandling={åpenBehandling} />
+                                    </VedtakBegrunnelserProvider>
+                                )}
                                 <VedtaksperioderMedBegrunnelser
                                     fagsak={fagsak}
                                     åpenBehandling={åpenBehandling}
+                                    erLesevisning={erLesevisning()}
                                 />
```

Comment:
Tenkte i utgangspunktet dette var en fornuftig måte å toggle på: https://github.com/navikt/familie-ba-sak-frontend/pull/1212/commits/3c5d0bda98ed29910468881ec4915883a0ebde1f 
men vi må støtte nytt format for fortsatt innvilget allerede, så må bare stole på at vi får rett fra backend (togglet der også)